project/security.py

```diff
@@ -0,0 +1,57 @@
+import sqlite3
+import hashlib
+
+class SecurityHandler:
+    def __init__(self):
+        try:
+            self.conn = sqlite3.connect('users.db')
+            self.create_users_table()
+        except sqlite3.Error as e:
+            print(e)
+    
+    def create_users_table(self):
+        '''
+        Creates tables of the database
+            if it already exists, it doesn't create it again
+        '''
+        c = self.conn.cursor()
+        c.execute('''
+            CREATE TABLE IF NOT EXISTS users (
+                username TEXT PRIMARY KEY
```

Comment:
@deionchaudhary1 

I think the SQL issue you are seeing is because this query is incorrect. You need a `,` after `username TEST PRIMARY KEY`

Would be good to check if the query executes correctly, and add some error handling here